app/forms/ApplicantLegalContactFormProvider.scala

```diff
@@ -21,23 +21,31 @@ import forms.mappings.Mappings
 import play.api.data.{Form, Forms}
 import play.api.data.Forms._
 import models.ApplicantLegalContact
+import play.api.i18n.Messages
 
 class ApplicantLegalContactFormProvider @Inject() extends Mappings {
 
-  val nameEmailLimit: Int = 200
-  val phonesLimit: Int    = 100
+  val nameEmailLimit: Int    = 200
+  val phonesLimit: Int       = 100
+  val rejectXssChars: String = """^[^<>"&]*$"""
```

Comment:
You're defining these same two regex vals in every form (and sometimes in tests) but I think it would be more maintainable to have them defined once in a helper object. Does that sound reasonable? Could be a new object or maybe throw them into `utils.CommonHelpers`?

Reply:
Yeah you're right, will make the changes